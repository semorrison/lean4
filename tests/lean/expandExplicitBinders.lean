namespace MySigma
  /-- We define a new `Sigma` -/
  def Sigma {α : Type u} (β : α → Type v)
    := String

  #reduce Σ a, a -- This should not reduce to `String`
end MySigma
