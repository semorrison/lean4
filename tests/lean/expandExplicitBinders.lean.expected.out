(a : Type u_1) Ã— a
